applications/GeoMechanicsApplication/custom_conditions/U_Pw_face_load_condition.cpp

```diff
@@ -107,6 +107,21 @@ void UPwFaceLoadCondition<2,3>::
     rIntegrationCoefficient = ds * Weight;
 }
 
+//----------------------------------------------------------------------------------------
+template< >
+void UPwFaceLoadCondition<2, 5>::
+CalculateIntegrationCoefficient(double& rIntegrationCoefficient,
+    const Matrix& Jacobian,
+    const double& Weight)
+{
+    double dx_dxi = Jacobian(0, 0);
```

Comment:
Thank you @AlejandroCornejo for your help. 
1. Related to your comment, I am working to add them to the Kratos core, and I am adding test cases too. It will be committed during this week, if everything goes smoothly.
 2. const is added